pycytominer/tests/test_cyto_utils/test_util.py

```diff
@@ -263,3 +245,59 @@ def test_extract_image_features():
     pd.testing.assert_frame_equal(
         expected_result.sort_index(axis=1), result.sort_index(axis=1)
     )
+
+
+def _test_pairwise_corr(data_df, expected_result):
```

Comment:
Thanks for refactoring out this function! I think this is a good idea.

I didn't immediately catch that it started with an underscore, can you update the name to make it more obvious that it is an internal function?

Perhapse `_assert_pairwise_corr_helper` or something like that? It'll also be good to add a one-line docstring to clarify.